projects/observability/src/pages/explorer/explorer.types.ts

```diff
@@ -0,0 +1,4 @@
+export const enum ScopeQueryParam {
```

Comment:
Why did I move this type to a separate file? To fix the circular structure of imports.

<img width="870" alt="Screenshot 2022-08-05 at 12 09 21 PM" src="https://user-images.githubusercontent.com/29686866/183020907-3f74101f-9be2-4e5b-8b9e-dbf148147205.png">

Ideally any exported types should be in separate files as a rule of thumb. Not sure why the Hypertrace team keeps everything in one file, leads to a lot of cases where 2 files are importing something from each other.